src/reconstruction/StandardModel.jl

```diff
@@ -1,47 +1,41 @@
 """
-    add!(model::StandardModel, rxns::Union{Vector{Reaction}, Reaction})
+    add_reactions!(model::StandardModel, rxns::Vector{Reaction})
 
-Add `rxn(s)` to `model` if they are not already present based on reaction `id`.
+Add `rxns` to `model` based on reaction `id`.
 """
-function add!(model::StandardModel, rxns::Vector{Reaction})
+function add_reactions!(model::StandardModel, rxns::Vector{Reaction})
     for rxn in rxns
-        add!(model, rxn)
+        model.reactions[rxn.id] = rxn
     end
 end
 
-function add!(model::StandardModel, rxn::Reaction)
-    model.reactions[rxn.id] = rxn
-end
+add_reactions!(model::StandardModel, rxn::Reaction) = add_reactions!(model, [rxn])
 
 """
-    add!(model::StandardModel, mets::Union{Vector{Metabolite}, Metabolite})
+    add_metabolites!(model::StandardModel, mets::Vector{Metabolite})
 
-Add `met(s)` to `model` if they are not already present, based on metabolite `id`.
+Add `mets` to `model` based on metabolite `id`.
 """
-function add!(model::StandardModel, mets::Vector{Metabolite})
+function add_metabolites!(model::StandardModel, mets::Vector{Metabolite})
     for met in mets
-        add!(model, met)
+        model.metabolites[met.id] = met
     end
 end
 
-function add!(model::StandardModel, met::Metabolite)
-    model.metabolites[met.id] = met
-end
-
+add_metabolites!(model::StandardModel, met::Metabolite) =  add_metabolites!(model, [met])
```

Comment:
ey, can we reduce the polymorphism here and call it just `add_metabolite!` ?

Reply:
lol, I actually had this but then second guessed myself. Will do!